test/any.specs.ts

```diff
@@ -0,0 +1,203 @@
+        !isEmpty(value)
+      })
+    ))
+
+    it("Should return true for null or undefined values", () => {
+      fc.assert(fc.property(fc.constant(null), (value) => isEmpty(value)));
+      fc.assert(fc.property(fc.constant(undefined), (value) => isEmpty(value)));
+    });
+
+  it("Should return true for false boolean", () => {
+    fc.assert(fc.property(fc.constant(false), (boo) => isEmpty(boo) === true))
+  })  
+
+  it("Should return false for true boolean", () =>
+  fc.assert(
+    fc.property(fc.constant(true), (boo) => !isEmpty(boo))
+  )
+);
+
+  it("Should return true for BigInt or Number values equal to 0", () =>
+  fc.assert(
+    fc.property(fc.oneof(fc.bigInt(), fc.integer()).filter(value => value === 0n),(value) => {
+        isEmpty(value) === (value === 0n)
+      }
+    )
+  ));
+
+  it("Should return false for BigInt or Number values not equal to 0", () =>
+  fc.assert(
+    fc.property(fc.oneof(fc.bigInt().filter(bigInt => bigInt !== 0n),fc.integer({ min: 1 })),(value) => 
+      !isEmpty(value)
+    )
+  ))
+
+  it("Should return true for empty Date", () =>
+    fc.assert(
+      fc.property(fc.constant(new Date(0)), (value) => isEmpty(value) === true)
+    )
+  );
+
+  it("Should return false for non-empty Date", () => {
+    fc.assert(
+      fc.property(fc.date({ min: new Date(1) }), (value: Date) => !isEmpty(value))
+    )
+  })
+
+  it("Should return true for empty Symbols", () => {
+    fc.assert(
+      fc.property(fc.anything(), (value) => {
+        if (typeof value === 'symbol') {
+          return isEmpty(value)
+        }
+      })
+    )
+  })
+
+  it("Should return false for non-empty Symbols", () => {
+    fc.assert(
+      fc.property(fc.anything(), (value) => {
+        if (typeof value === 'symbol') {
+          return !isEmpty(value);
+        }
+      })
+    )
+  })
+})
+
+//isNotEmpty
+describe("isNotEmpty: ", () => {
+  it("Should return true for non-empty strings", () => {
+    fc.assert(
+      fc.property(fc.string({ minLength: 1 }), (value) => {
+        return isNotEmpty(value)
+      })
+    )
+  })
+
+  it("Should return false for empty strings", () => {
+    fc.assert(
+      fc.property(fc.string({ maxLength: 0 }), (value) => {
+        return !isNotEmpty(value);
+      })
+    );
+  });
+})
+
+//areEquals
+describe("areEquals: ", () => {
+  it("Should return true for equal values", () => {
+    fc.assert(
+      fc.property(fc.anything(), (value) => {
+        return areEquals(value, value)
+      })
+    )
+  })
+
+  it("Should return false for different values", () => {
+    fc.assert(
+      fc.property(fc.anything(), fc.anything(), (value1, value2) => {
+        fc.pre(value1 !== value2);
+        return !areEquals(value1, value2)
+      })
+    )
+  })
+
+// Définir un comparateur personnalisé pour les objets ayant les mêmes clés et valeurs, mais de types différents
+const sameKeysAndValuesArbitrary = fc
+  .dictionary(fc.string(), fc.oneof(fc.string(), fc.integer()))
```

Comment:
Also, you re limiting yourself to string keys and string or number values. Why ?